design-proposals/enhancements.md

```diff
@@ -0,0 +1,108 @@
+# Overview
+
+Rework the existing KubeVirt design proposal process into a more
+formal - but still lean - and SIG-aligned Enhancement Proposal process.
+
+## Motivation
+
+Today it's a [very small group](https://github.com/kubevirt/community/blob/main/OWNERS#L7-L14)
+which can approve design-proposals.
+While certain individuals can approve design proposals, once they are merged
+it is not clear who is owning them post-merge.
+
+In this proposal we are proposing to change the process and shape them
+around SIGs.
+
+At its core, this proposal requires:
+
+1. designs to be sponsored by a SIG
+2. to make one SIG responsible for the design process (reviews of design, code, documentation)
+3. to make one SIG responsible for managing the design process (collaboration with other SIGs as needed)
+4. to make one SIG to own the feature once it has been merged. The SIG is responsible for maintaining, fixing, running, _everything-it_.
+
+This has a few effects - see the following Goals section.
+
+## Goals
+
+1. The design process now has a mechanism to distribute designs among SIGs
+2. SIG approvers are empowered to approve designs, increase the approvers pool
+3. The ownership of an implemented feature becomes clear
+4. Ensure that designs converge (accept, reject)
+5. Formalize this process as an Virtualization Enhancement Proposal (VEP) process
+
+## Non Goals
+
+1. Create unnecessary paperwork
+
+## Definition of Users
+
+* VEP Author - The person writing a design/enhancement proposal
+* SIG - A formal KubeVirt SIG
+* SIG Approver - A member of a SIG with approval permissions
+
+## User Stories
+
+* As a VEP Author I want to know who I can work with in order to move
+  my proposal forward
+* As a SIG I want to have a say in what is getting pushed into my domain
+  in order to make sure that we are able to maintain it
+* As a SIG Approver I want to ensure that a design is sound before a
+  VEP Author is approaching an implementation.
```

Comment:
Maybe add:
* As a Kubevirt developer, I want to have a single source-of-truth w.r.t. the current status, design API of a feature.

(see my comment below regarding this topic)

Reply:
Perect, agreed, added.